index.html

```diff
@@ -222,7 +214,7 @@ <h2>Securing the VC Data Model</h2>
     <p>
       The most specific media type (or subtype) available SHOULD be used, instead of
       more generic media types (or supertypes). For example, rather than the general
-      <code>application/sd-jwt</code>, <code>application/vc+ld+json+sd-jwt</code>
+      <code>application/sd-jwt</code>, <code>application/vc2+sd-jwt</code>
```

Comment:
```suggestion
      <code>application/sd-jwt</code>, <code>application/vc+...+sd-jwt</code>
```

Reply:
@selfissued would this address your concern? I would like to have a PR that leaves the door open to removing the reference to multiple suffixes entirely.

Reply:
What does the ... mean?

Reply:
it means, not relevant to understanding the paragraph.